From eda94122869e0e33a9b8f54e522c85919b5a00db Mon Sep 17 00:00:00 2001
From: Lu Yahan <yahan@iscas.ac.cn>
Date: Wed, 12 May 2021 09:20:19 +0800
Subject: [PATCH] [riscv64] Skip incompatible cctests

In debug mode, these will currently cause a DCHECK failure or a
segmentation fault.

See also: https://github.com/riscv/v8/issues/490

Change-Id: I2a4b8e0c9b0fb85393b41b1016b2caa2f013bcc4
Reviewed-on: https://chromium-review.googlesource.com/c/v8/v8/+/2881505
Commit-Queue: Yahan Lu <yahan@iscas.ac.cn>
Reviewed-by: Jakob Gruber <jgruber@chromium.org>
Cr-Commit-Position: refs/heads/master@{#74550}
---
 test/cctest/cctest.status | 11 ++++++++++-
 1 file changed, 10 insertions(+), 1 deletion(-)

diff --git a/test/cctest/cctest.status b/test/cctest/cctest.status
index 820748d8fd..31ffa5b466 100644
--- a/test/cctest/cctest.status
+++ b/test/cctest/cctest.status
@@ -401,7 +401,16 @@
   'test-run-wasm-atomics/*': [SKIP],
   'test-run-wasm-64/*': [SKIP],
   'test-run-wasm/*': [SKIP],
-}],
+
+  # https://github.com/riscv/v8/issues/490
+  'test-api/CallICFastApi_TrivialSignatureWithProfiler': [SKIP],
+  'test-api/FunctionTemplateWithProfiler': [SKIP],
+  'test-api/InterceptorCallICFastApi_TrivialSignatureWithProfiler': [SKIP],
+  'test-api/LoadICFastApi_DirectCall_GCMoveStubWithProfiler': [SKIP],
+  'test-api/SimpleCallbackWithProfiler': [SKIP],
+  'test-cpu-profiler/*': [SKIP],
+
+}],  # 'arch == riscv64'
 
 ##############################################################################
 ['arch == riscv64 and simulator_run', {
-- 
2.35.1

